nms/shared/types/network.ts

```diff
@@ -50,4 +50,19 @@ export type NetworkId = string;
 export type NetworkName = string;
 export type PolicyId = string;
 export type SubscriberId = string;
+export type Tier = {
```

Comment:
We should use `Tier` from `api.ts`

Reply:
Removed the custom Tier type again. Used `string` instead of `TierID`.